lib/commands/publish-ios.ts

```diff
@@ -0,0 +1,46 @@
+///<reference path="../.d.ts"/>
+"use strict";
+
+import {StringCommandParameter} from "../common/command-params";
+
+export class PublishIOS implements ICommand {
+	constructor(private $injector: IInjector,
+		private $itmsTransporterService: IITMSTransporterService,
+		private $logger: ILogger,
+		private $options: IOptions,
+		private $prompter: IPrompter,
+		private $stringParameterBuilder: IStringParameterBuilder) { }
+
+	public allowedParameters: ICommandParameter[] = [this.$stringParameterBuilder.createMandatoryParameter("Missing required Apple ID parameter."),
```

Comment:
I'm against declaring this as mandatory parameter. We can prompt the user for it as well. Also can we validate that such application exists in iTunes (appbuilder-cli has some logic for this)